Index: baloo-4.13.0/src/pim/search/email/emailsearchstore.cpp
===================================================================
--- baloo-4.13.0.orig/src/pim/search/email/emailsearchstore.cpp	2014-04-09 12:33:16.000000000 +0200
+++ baloo-4.13.0/src/pim/search/email/emailsearchstore.cpp	2014-04-15 11:45:22.754275526 +0200
@@ -105,7 +105,7 @@
     try {
         data = doc.get_data();
     }
-    catch (const Xapian::DatabaseError&) {
+    catch (const Xapian::Error&) {
         // Nothing to do, move along
     }
 
